stdlib/did_you_mean/0/did_you_mean.rbs

```diff
@@ -0,0 +1,348 @@
+    # reserved words require an argument, and a `NoMethodError` is raised due to the
+    # presence of the argument.
+    #
+    # The `MethodNameChecker` will use this list to suggest a reversed word if a
+    # `NoMethodError` is raised and found closest matches.
+    #
+    # Also see `VariableNameChecker::RB_RESERVED_WORDS`.
+    #
+    RB_RESERVED_WORDS: Array[Symbol]
+
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/method_name_checker.rb
+    #   - new(exception)
+    # -->
+    #
+    def initialize: (NoMethodError[untyped] exception) -> void
+
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/method_name_checker.rb
+    #   - corrections()
+    # -->
+    #
+    def corrections: () -> Array[Symbol]
+  end
+
+  class NullChecker
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/null_checker.rb
+    #   - new(*)
+    # -->
+    #
+    def initialize: (*untyped) -> void
+
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/null_checker.rb
+    #   - corrections()
+    # -->
+    #
+    def corrections: () -> Array[untyped]
+  end
+
+  class PatternKeyNameChecker
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/pattern_key_name_checker.rb
+    #   - new(no_matching_pattern_key_error)
+    # -->
+    #
+    def initialize: (untyped) -> void
+
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/pattern_key_name_checker.rb
+    #   - corrections()
+    # -->
+    #
+    def corrections: () -> Array[String]
+  end
+
+  class RequirePathChecker
+    INITIAL_LOAD_PATH: Array[String]
+    ENV_SPECIFIC_EXT: String
+
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/require_path_checker.rb
+    #   - requireables()
+    # -->
+    #
+    def self.requireables: -> Array[String]
+
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/require_path_checker.rb
+    #   - new(exception)
+    # -->
+    #
+    def initialize: (untyped exception) -> void
+
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checkers/require_path_checker.rb
+    #   - corrections()
+    # -->
+    #
+    def corrections: () -> Array[String]
+  end
+
+  class SpellChecker
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checker.rb
+    #   - new(dictionary:)
+    # -->
+    #
+    def initialize: (dictionary: Array[String | Symbol]) -> void
+
+    # <!--
+    #   rdoc-file=lib/did_you_mean/spell_checker.rb
+    #   - correct(input)
+    # -->
+    #
+    def correct: (String | Symbol input) -> Array[String]
+  end
+
+  # <!-- rdoc-file=lib/did_you_mean/tree_spell_checker.rb -->
+  # spell checker for a dictionary that has a tree structure, see
+  # doc/tree_spell_checker_api.md
+  #
+  class TreeSpellChecker
+    # <!--
+    #   rdoc-file=lib/did_you_mean/tree_spell_checker.rb
+    #   - new(dictionary:, separator: '/', augment: nil)
+    # -->
+    #
+    def initialize: (dictionary: Array[String], ?separator: String, ?augment: bool?) -> void
```

Comment:
> __[Goodcheck]__ Prefer `boolish` over `bool` for method arguments and block return values ([view](https://sider.review/gh/repos/174796313/pulls/980#issue-6870548))
>
> <details>
>
> See the doc below:
> https://github.com/ruby/rbs/blob/78d04a2db0f1c4925d2b13c2939868edf9465d6c/docs/syntax.md#bool-or-boolish
>
> | Rule |
> | ---- |
> | `rbs.prefer_boolish` |
>
> You can [close this issue](https://sider.review/gh/repos/174796313/pulls/980#issue-6870548) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When you strictly want `true | false`. Use `bool` in this case.